source/reference/minio-mc/mc-support-register.rst

```diff
@@ -0,0 +1,82 @@
+=======================
+``mc support register``
+=======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support register
+
+Description
+-----------
+
+The :mc-cmd:`mc support register` command connects your deployment with your |SUBNET| account.
+
+After registration, upload deployment health reports directly to SUBNET using :mc-cmd:`mc support diagnostics` command.
+
+
+Examples
+--------
+
+Register a Cluster Using the Cluster's Name
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Register the MinIO cluster at alias ``minio1`` on SUBNET, using ``minio1`` as the cluster name:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support register play
+
+Register a Cluster with a Different Cluster Name
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Register a MinIO cluster at alias ``minio2`` on SUBNET, using ``second-cluster`` as the name:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support register minio2 --name second-cluster
+
+Syntax
+------
+      
+The command has the following syntax:
+
+.. code-block:: shell
+
+   mc [GLOBALFLAGS] support register       \
+                            ALIAS          \
+                            [--name value] \
+                            [--airgap]
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: ALIAS
+   :required:
+
+   The :ref:`alias <alias>` of the MinIO deployment.
+
+.. mc-cmd:: --name
+   :optional:
+
+   Specify a name other than the alias to associate to the MinIO cluster in SUBNET.
+
+   Use ``--name <value>`` replacing ``<value>`` with the name you want to use for the cluster on SUBNET.
+   
+.. mc-cmd:: --airgap
+   :optional:
+
+   Use in environments without network access to SUBNET (for example, airgapped, firewalled, or similar configuration).
```

Comment:
Maybe we should explain how the `--airgap` flag works, as it is not very straight forward. Running the command with `--airgap` shows a registration token to the user and waits on the terminal.

User then has to paste it on the relevant screen in SUBNET. SUBNET then displays the api key for the registered cluster, which needs to be pasted back on the terminal to complete the registration flow.

Reply:
Added a procedure to the --airgap flag.